test/image/mocks/zzz-automargin-title-container-bottom.json

```
@@ -0,0 +1,30 @@
+{
+    "data": [
+        {
+            "showlegend": false,
+            "type": "scatter",
+            "x": [
+            1,
+            2,
+            3
+            ],
+            "y": [
+            4,
+            5,
+            6
+            ]
+        }],
+    "layout": {
+        "height": 300,
+        "width": 400,
+        "margin": {"t":0, "b": 0, "l": 0, "r": 0},
+        "xaxis": {"automargin": true, "title": {"text": "x-axis title"}},
+        "title": {
+            "automargin": true, 
+            "text": "Basic title",
```

Thanks @hannahker for adding new tests.
Now let's use `container` and `paper` somewhere within the `title.text` so that the tests become self describing.
I also used a descender like `p` in the text.
```suggestion
            "text": "Container | no-pad",
```